terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -92,11 +121,12 @@ class Input:
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **kwargs):
+        log = audit_log_services.new_audit_logger()
```

Comment:
suggestion: Same thing as the other method, it would be great to have this as a class variable in the parent class.